ppr-ui/src/components/common/AccountInfo.vue

```diff
@@ -93,6 +96,11 @@ export default defineComponent({
     accountInfo: {
       type: Object as () => AccountInfoIF,
       required: true
+    },
+    desc: {
+      type: String,
+      default: null,
+      required: false
```

Comment:
Not 'required' .....  😁 (see what i did there?) 

Just messing around, i believe by default this property is false so doesn't necessarily need an explicit `false`. Doesn't hurt though either!